flink-rpc/flink-rpc-akka/src/test/java/org/apache/flink/runtime/rpc/akka/TimeoutCallStackTest.java

```diff
@@ -93,10 +91,12 @@ public void testTimeoutException() throws Exception {
             failureCause = e.getCause();
         }
```

Comment:
It looks like the first part of the test is no longer necessary -- we're calling `.get` twice.